specs/TrackingPrevention.md

```diff
@@ -0,0 +1,197 @@
+        }
+    }
+    catch (NotImplementedException exception)
+    {
+        MessageBox.Show(this,
+           "Failed to set tracking prevention user preference: " +
+           exception.Message, "Tracking Prevention User Preference");
+    }
+}
+```
+
+```cpp
+void SettingsComponent::SetTrackingPreventionUserPreference(
+    COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND value)
+{
+    wil::com_ptr<ICoreWebView2_13> webView2_13;
+    webView2_13 = m_webView.try_query<ICoreWebView2_13>();
+
+    if (webView2_13)
+    {
+        wil::com_ptr<ICoreWebView2Profile> profile;
+        CHECK_FAILURE(webView2_13->get_Profile(&profile));
+
+        auto profile5 = profile.try_query<ICoreWebView2Profile5>();
+        if (profile5)
+        {
+            profile5->put_TrackingPreventionUserPreference(value);
+        }
+    }
+}
+```
+
+# API Details
+```
+/// Tracking prevention user preference kind.
+[v1_enum] typedef enum COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND {
+  /// Indicates that tracking prevention is turned off. Setting to this level if
+  /// `IsTrackingPreventionEnabled` is true means privacy protection features are enabled
+  /// but will not block cookie and resource requests from known tracking domains.
+  COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_OFF,
+  /// The least restrictive level of tracking prevention for users who enjoy
+  /// personalized advertisements and who do not mind being tracked on the web.
+  /// This only protects users against malicious trackers such as fingerprints and crypto miners.
+  COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_BASIC,
+  /// The default level of tracking prevention for users who want to see less creepy
+  /// advertisements that follow them around the web while they browse. Balanced aims to
+  /// block trackers from sites that users never engage with while minimizing the risk of
+  /// compatibility issues on the web.
+  COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_BALANCED,
+  /// The most restrictive level of tracking prevention for users who are okay
+  /// trading website compatibility for maximum privacy.
+  COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_STRICT,
+} COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND;
+
+/// Additional options used to create WebView2 Environment.
+[uuid(12e494a2-c876-11eb-b8bc-0242ac130003), object, pointer_default(unique)]
+interface ICoreWebView2EnvironmentOptions3 : IUnknown {
+  /// The `IsTrackingPreventionEnabled` property is used to toggle tracking prevention feature in WebView2.
+  /// By default this feature is enabled which means privacy protection features are enabled that blocks cookies
+  /// and web requests from known tracking domains and set to
+  /// `COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_BALANCED`.
+  /// You can set this property to false to disable the tracking prevention feature that also skips the
+  /// related code and improves the performance.
+  ///
```

Comment:
We should also give a recommendation of why they might want to disable the feature (the app only renders content in the WebView2 that is known to be safe and they want to improve runtime performance) and when they shouldn't disable it (the app renders arbitrary web content in the WebView2 and should protect end user privacy).